guide/src/web-getting-started.md

```diff
@@ -1,9 +1,9 @@
 # Getting Started on the Web
 
-This section of the guide covers how to integrate Ferrostar into a web app.
+This section of the guide covers how to integrate **Ferrostar** into a web app using **React**.
```

Comment:
This is oddly specific; we shouldn't make the setup exclusive to React. I think we can leave the original language and get into specifics later in the document since most things are common web standards.

Reply:
Sorry, I misunderstood about the changes, and made React Specific.
Ok i am going to undo these un-necessary changes.